smart_sa/intervention/models.py

```diff
@@ -936,11 +936,10 @@ def relevant_timestamps(self, session):
                 self.session_activity_timestamps(session))
 
     def session_timestamps(self, session):
-        sv_set = self.sessionvisit_set.all()
-        if not sv_set:
+        sv_set = self.sessionvisit_set.all().filter(session_id=session.id)
```

Comment:
Can remove the `.all()` here, like so: `sv_set = self.sessionvisit_set.filter(session_id=session.id)`

Reply:
Got it. I fixed it.  I just amended the commit and force pushed, in case you have it checked out locally.